### ISSUE TITLE
Tvheadend issues

### ISSUE THREAD

**Issue by jjLDN** [SynoCommunity/spksrc#160]
Hi.
Tried that spk yestarday.
I have some issues though.
I don't know how to use it.
I cannot choose any grabber in xml tv. The window disappears immediately. I cannot choose any tv card even though I have two different devices an both recognizable by DSM.
In IPTV section I cannot add any proper tv stream from internet.
Whenever I tried to stop the package I cannot turn it on back again. It appears to be hanging in ram and not closing properly.
So basically I don't know what is the purpose of this package.
I use DS712+ on DSM 4.0-2228.
